cracked_hash_stats.py

```diff
@@ -22,7 +22,7 @@ def runstats(hcoutput, ntdsdump):
     crackedpws = []
     # Determine the number of unique hashes processed by placing all ntds dump
     # lines in a set.
-    for dumpline in ntdsdump:
```

Comment:
I assume this was just a find/replace gone wrong or something?

 'f' is not defined anywhere, so this won't run.